changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
+  - app_icons
+  - apple_silicon_security_policy
+  - falconctl_options
+  - falcon_kernel_check
+  - cryptoinfo
+  - cryptsetup_status
+  - parse_plist
+  - parse_json
+  - parse_jsonl
+  - parse_xml
+  - parse_ini
+  - dev_table_tooling
+  - dsim_default_associations
+  - filevault_status
+  - firefox_preferences
+  - firmwarepasswd
+  - gsettings
+  - gsettings_metadata
+  - ioreg
+  - secedit
+  - system_profiler
+  - windows_updates
+  - wmi
+  - xfconf
+  - xrdb
+  - zfs_properties
+  - zpool_properties
```

Comment:
```suggestion
```

I don't know what value these tables provide. I haven't heard a customer talk about them. Let's cut these for now unless folks think otherwise. We can add them later.